networks/fabric/v2/v2.1.0/2org1peergoleveldb_raft/api/fabric-api-solo-go-prometheus.yaml

```diff
@@ -71,10 +71,10 @@ channels:
                 eventSource: true
 
         chaincodes:
-            - id: fixed-asset
+            - id: fixed-asset-base
               version: v0
               language: golang
-              path: fabric/api/fixed-asset/go
+              path: fabric/api/fixed-asset-base/go
               metadataPath: src/fabric/api/fixed-asset/metadata
```

Comment:
Chaincode metadataPath needs to be changed to `src/fabric/api/fixed-asset-base/metadata`